examples/record-playback/rs-record-playback.cpp

```diff
@@ -147,7 +147,7 @@ int main(int argc, char * argv[]) try
                     pipe->stop(); // Stop streaming with default configuration
                     pipe = std::make_shared<rs2::pipeline>();
                     rs2::config cfg;
-                    cfg.enable_device_from_file("a.bag");
+                    cfg.enable_device_from_file("a.bag", false);
```

Comment:
Do you want to disable repeat in the demo, or this line was just for testing and got into the PR?